src/olympia/users/templates/users/mobile/fxa_migration.html

```diff
@@ -0,0 +1,25 @@
+{% extends "mobile/base.html" %}
+
+{% block title %}{{ page_title(_('Migrate to Firefox Accounts')) }}{% endblock %}
+
+{% block page %}
+
+<div class="primary fxa-migration-prompt" role="main">
+  <h2>{{ _('Migrate to Firefox Accounts') }}</h2>
+  <img alt="{{ request.APP.pretty }}" src="{{ static('img/icons/firefox.png')|safe }}" class="firefox-logo">
+  <p>
+    {% trans %}
+      Mozilla Add-ons has transitioned to Firefox Accounts for login.
+      Continue to complete the simple upgrade process.
+    {% endtrans %}
+  </p>
+  <div class="migrate-button">
+    <a href="{{ url('users.login') }}" class="fxa-login button prominent">
+      {{ _('Continue') }}
+    </a>
+  </div>
+  <div class="skip-migrate-button">
+    <a href="{{ to }}">{{ _('Skip') }}</a>
+  </a>
+</div>
+{% endblock page %}
```

Comment:
this template file looks like a duplication of the [desktop one](https://github.com/mozilla/addons-server/blob/master/src/olympia/users/templates/users/fxa_migration.html). Can't you just make a common include that both templates can share for the content?